notebook/conversation.ipynb

```diff
@@ -0,0 +1,187 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b75c96fc-e399-4f81-bcf8-8a5d15bcb79b",
+   "metadata": {},
+   "source": [
+    "# Conversation with Agent"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25d30c19de76e93f",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "In this notebook, we will show a demo of how to program a multi-agent conversation in AgentScope.Complete code is in `examples/conversation/conversation.py`, which sets up a user agent and an assistant agent to have a conversation. When user input \"exit\", the conversation ends. You can modify the sys_prompt to change the role of assistant agent."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28c4c4dc-7107-45e5-9ef4-f6506a0d55e2",
+   "metadata": {},
+   "source": [
+    "To install AgentScope, you need to have Python 3.9 or higher installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6dbd5990-ed46-4727-84b1-d4131c5a6245",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install AgentScope"
```

Comment:
Follow README.md to install AgentScope.

Reply:
Do you mean just write "Follow README.md to install AgentScope." or copy the codes from README.md here?